Gemfile

```diff
@@ -4,27 +4,24 @@ source "https://rubygems.org"
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
+gem "base64"
 gem "canister"
 gem "dotenv"
 gem "ettin"
 gem "faraday"
 gem "push_metrics", git: "https://github.com/hathitrust/push_metrics.git", tag: "v0.9.1"
 gem "mongo"
-gem "mongoid"
+gem "mongoid", "~> 8.1"
 gem "mysql2"
+gem "pry"
```

Comment:
we require pry for `phctl pry` so I figured it should not be in the "development" group only